packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx

```diff
@@ -52,6 +54,8 @@ export const ShowPageRightContainer = ({
   notes,
   emails,
 }: ShowPageRightContainerProps) => {
+  const isMessagingEnabled = useIsFeatureEnabled('IS_MESSAGING_ENABLED');
```

Comment:
Can we implement a string literal or enum for those feature flags ?

Reply:
Done! I added the `FeatureFlagKey` type.